production/packaging/azure/seller_frontend_service/bin/envoy.yaml

```diff
@@ -105,7 +105,7 @@ static_resources:
             address:
               socket_address:
                 address: 0.0.0.0
-                port_value: 50051
+                port_value: 50053
```

Comment:
Is this the port Envoy listens on? Is it an issue that 50053 is the port the buyer frontend listens on or is that deliberate?